bin/deploy.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+
+
+# list of dependencies to be check
+dependencies=(
+    "@taqueria/protocol"
+    "@taqueria/node-sdk"
+)
+
+packages=(
+    "taqueria-protocol"
+    "taqueria-sdk"
+    "taqueria-plugin-flextesa"
+    "taqueria-plugin-contract-types"
+    "taqueria-flextesa-manager"
+    "taqueria-plugin-ipfs-pinata"
+    "taqueria-plugin-jest"
+    "taqueria-plugin-smartpy"
+    "taqueria-vscode-extension"
+    "taqueria-plugin-ligo"
+    "taqueria-plugin-mock"
+    "taqueria-plugin-archetype"
+    "taqueria-state"
+    "taqueria-plugin-tezos-client"
+    "taqueria-plugin-taquito"
+    "taqueria-utils"
+)
+
+echo "Runng \"npm version -ws $1\""
```

Comment:
Just need to check the spelling of the word "running" in some of these commands

Reply:
Fixed